src/scrapers/fass_scraper/scraper.py

```diff
@@ -1,38 +1,30 @@
-
-PRODUCT_ID_INDEX = 5
-
-
 # Scarping method for extracting the bipacksedel subcategory on fass.se
 # Example page: https://www.fass.se/LIF/product?userType=2&nplId=20190822000136&docType=7&scrollPosition=352
-def extract_medical_text(soup, search_tag, recursive, nested):
+def extract_medical_text(soup):
     data = {}
-    # Select full div containing the medical text
+    
     fass_content = soup.select('.fass-content')[0]
-
     # Every section is labled with an a tag defining what information followes
     #   used as index for our data dictionary
-    headers = fass_content.find_all(search_tag, recursive=recursive)
+    headers = fass_content.find_all("h2", recursive=True)
 
     for section in headers:
         try:
-            # Get the title from the current section
-            if not nested:
-                title = section.get('id')
-            else:
-                title = section.find('a', id=True).get('id')
+            title = section.find('a', id=True).get('id')
         except:
             continue
-    
+
         # Collect all html between two <a> tags
```

Comment:
fix: change to h2-tag
```suggestion
        # Collect all html between two <h2> tags
```

Reply:
Accepted